config/main.py

```diff
@@ -1252,14 +1308,8 @@ def add(ctx, interface_name, ip_addr, gw):
 
     try:
         ipaddress.ip_network(unicode(ip_addr), strict=False)
-        if interface_name.startswith("Ethernet"):
-            if VLAN_SUB_INTERFACE_SEPARATOR in interface_name:
-                config_db.set_entry("VLAN_SUB_INTERFACE", interface_name, {"admin_status": "up"})
-                config_db.set_entry("VLAN_SUB_INTERFACE", (interface_name, ip_addr), {"NULL": "NULL"})
-            else:
-                config_db.set_entry("INTERFACE", (interface_name, ip_addr), {"NULL": "NULL"})
-                config_db.set_entry("INTERFACE", interface_name, {"NULL": "NULL"})
-        elif interface_name == 'eth0':
+
+        if interface_name == 'eth0':
```

Comment:
suggest to don't hardcode here.

Reply:
It has nothing to do with this PR, it is just merged from master. If to change, a new PR is needed. I think mgmt intf should be handled in another command.